graylog2-server/src/main/java/org/graylog2/contentpacks/model/constraints/Constraint.java

```diff
@@ -28,4 +30,6 @@
 public interface Constraint extends Typed {
     interface ConstraintBuilder<SELF> extends Typed.TypeBuilder<SELF> {
     }
+
+    Optional<Boolean> fulfilled();
```

Comment:
The `fulfilled` field should be removed from `Constraint` and all classes that implement `Constraint`.